src/inverseRobot.js

```diff
@@ -24,7 +24,21 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  let i = 0;
```

Comment:
let's use more meaningful variables names.
If it is the count of robot parts, name it smth like 'counter'. Name 'i' mean's nothing.